charon.py

```diff
@@ -192,14 +194,29 @@ async def charon_help(message):
 async def on_message(message):
     # Messages that should bring up the "unrecognized command" reply: ".foo", "."
     # Messages that should not: "..", "..."
-    if message.content.startswith(COMMAND_PREFIX):
-        command = message.content.split(" ")[0][1:]
-        if command in LIST_OF_COMMANDS.keys():
-            # See above for command functions
-            await LIST_OF_COMMANDS[command](message)
-        elif not command.startswith(COMMAND_PREFIX):
-            await message.channel.send(
-                f'{message.author.name}, that is an unrecognized command. '
-                f'For a list of supported commands, please send \"{COMMAND_PREFIX}help\"')
+    result = re.match('^\.([a-zA-Z0-9]+)', message.content)
```

Comment:
I think the regex should use the command prefix:
```suggestion
    result = re.match(f'^{COMMAND_PREFIX}([a-zA-Z0-9]+)', message.content)
```

Reply:
Oh right, major oversight! Thanks!